traits-stubs/setup.py

```diff
@@ -0,0 +1,24 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+import setuptools
+
+
+if __name__ == "__main__":
+    setuptools.setup(
+        name="traits-stubs",
+        version="0.1.0",
+        description="type annotation integration stubs for traits",
+        install_requires=["mypy", "traits"],
+        packages=["traits-stubs"],
```

Comment:
I think we should include the "traits-stubs/tests" package, too (we normally distribute tests as part of a package).